src/libraries/System.Console/src/System.Console.csproj

```diff
@@ -3,6 +3,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Android;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <NoWarn Condition="'$(TargetsBrowser)' == 'true'">$(NoWarn);CA1416</NoWarn>
```

Comment:
What is this about?

Reply:
These are related to this issue https://github.com/dotnet/runtime/issues/44231, these are bugs with the sdk.

Reply:
As mentioned in the other thread, why not just disable the functionality in the sdk that adds these attributes? I don't think we should regress and fix this here instead.

Reply:
Its not a regression and more likely a scenario they should address. But i think if we could disable it for the adding these attributes for this project only that would be awesome.
I dont think we should disable it for the entire repo

Reply:
we discussed this offline right now. please disable the sdk behavior and file an issue in dotnet/runtime to discuss if we should use the sdk's feature instead of us adding the attributes ourselves.